pkg/deploy/identity-provider/identity_provider.go

```diff
@@ -121,6 +121,12 @@ func syncKeycloakResources(deployContext *deploy.DeployContext) (bool, error) {
 				break
 			}
 		}
+		_, err := util.K8sclient.ExecIntoPod(
```

Comment:
Here is problem. This script is run on every reconcile loop, so operator log is flooded with:
```
time="2021-04-21T12:02:38Z" level=info msg="Running exec for 'Update redirect URI-s' in the pod 'keycloak-cbd645d95-2l85z'"
time="2021-04-21T12:02:44Z" level=info msg="Exec successfully completed."
time="2021-04-21T12:02:47Z" level=info msg="Running exec for 'Update redirect URI-s' in the pod 'keycloak-cbd645d95-2l85z'"
time="2021-04-21T12:02:51Z" level=info msg="Exec successfully completed."
time="2021-04-21T12:02:57Z" level=info msg="Running exec for 'Update redirect URI-s' in the pod 'keycloak-cbd645d95-2l85z'"
time="2021-04-21T12:03:01Z" level=info msg="Exec successfully completed."
```

To provision keycloak resources once we have `Status.KeycloakProvisoned` field, but for this case we can introduce some local variable to update keycloak once.

Reply:
strange, i see it only once:
![image](https://user-images.githubusercontent.com/1651062/115559966-d7065580-a2bc-11eb-95f1-26efa856fab9.png)

Reply:
That's because of updating.
But in case of deploying from the scratch you will see many those lines